### DISCUSSION TITLE
Не качает с 1.readmanga.ru, 1.readmanga.io

### DISCUSSION THREAD

**Discussion by Sakuramory** [lirix360/ReadmangaGrabber#103] [Q&A]
При попытке скачать с  "1.readmanga.ru", "1.readmanga.io"  выдает ошибку "Указанный вами адрес не поддерживается."

**Comment by lirix360** [comment]
> При попытке скачать с "1.readmanga.ru", "1.readmanga.io" выдает ошибку "Указанный вами адрес не поддерживается."

Адреса добавлены
